testutil/e2e/protocolE2E.go

```diff
@@ -1165,31 +1161,17 @@ func (lt *lavaTest) getKeyAddress(key string) string {
 
 func calculateProviderCU(pairingClient pairingTypes.QueryClient) (map[string]uint64, error) {
 	providerCU := make(map[string]uint64)
-	paymentStorageClientRes, err := pairingClient.UniquePaymentStorageClientProviderAll(context.Background(), &pairingTypes.QueryAllUniquePaymentStorageClientProviderRequest{})
+	res, err := pairingClient.ProviderEpochCu(context.Background(), &pairingTypes.QueryProviderEpochCuRequest{})
```

Comment:
does this return all the providers?

Reply:
yes. changed the query name here: https://github.com/lavanet/lava/commit/3e4937ad2fc531aae9d346db710b69820845ff4e